modules/nf-core/gatk4/determinegermlinecontigploidy/main.nf

```diff
@@ -3,9 +3,6 @@ process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
     label 'process_single'
 
 
-    if(params.enable_conda){
```

Comment:
Does the warning need to be removed entirely, or the condition to check the conda scope

`if (session.config.conda && session.config.conda.enable){`

Reply:
I'd prefer this indeed

Reply:
Oooh awesome! Nvm my comment then 😁

Reply:
Ah oops, All good!